web-config.js

```diff
@@ -0,0 +1,21 @@
+const { sortMemberRules } = require('./sort-member-config');

```

Comment:
Sorry I didn't look closely enough... we actually already have a `browser-config` that I think was meant to fill the same purpose. Shall we just move all this into there and switch Lit/Polymer over to use it?